packages/supi/src/link.ts

```diff
@@ -209,13 +210,17 @@ async function linkToModules (
     prefix: string,
   },
 ) {
-  const dest = path.join(opts.destModulesDir, opts.alias)
-  const {reused} = await symlinkDir(opts.packageDir, dest)
+
+  const destModulesDirReal = fs.realpathSync(opts.destModulesDir)
+  const packageDirReal = fs.realpathSync(opts.packageDir)
```

Comment:
use `import fs = require('mz/fs')`

and the async version `await fs.realpath(...`